apps/api/src/feedback/controller/feedback.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller, Post, Body } from '@nestjs/common'
+import { Public } from '../../decorators/public.decorator'
+import { FeedbackService } from '../service/feedback.service'
+import { ApiTags } from '@nestjs/swagger'
+
+@ApiTags('Feedback Controller')
+@Controller('feedback')
+export class FeedbackController {
+  constructor(private readonly feedbackService: FeedbackService) {}
+
+  @Public()
+  @Post()
```

Comment:
Would you please add the swagger docs annotations? You can find them in `auth.controller.ts`. Basically, we will need these:
- request body type
- response categories
- authentication details